### ISSUE TITLE
Documentation: --freeze-backbone

### ISSUE THREAD

**Issue by jbarth-ubhd** [mittagessen/kraken#739]
From https://kraken.re/main/training/rectrain.html : »If the new dataset is fairly dissimilar or your base model has been pretrained with ketos pretrain, use --warmup in conjunction with --freeze-backbone for one 1 or 2 epochs.«

But later on that page, the following command line is given in the section »Unsupervised recognition pretraining«:
`ketos train -i pretrain_best.mlmodel --warmup 5000 --freeze-backbone 1000 ...`

Is »Unsupervised recognition pretraining« a complete different use case, so »1000« instead of »1 or 2« ?

Kind regards, Jochen

**Comment by mittagessen** [comment]
On 25/10/21 04:41AM, jbarth-ubhd wrote:
> jbarth-ubhd created an issue (mittagessen/kraken#739)
> 
> From https://kraken.re/main/training/rectrain.html : »If the new dataset is fairly dissimilar or your base model has been pretrained with ketos pretrain, use --warmup in conjunction with --freeze-backbone for one 1 or 2 epochs.«
> 
> But later on that page, the following command line is given in the section »Unsupervised recognition pretraining«:
> `ketos train -i pretrain_best.mlmodel --warmup 5000 --freeze-backbone 1000 ...`
> 
> Is »Unsupervised recognition pretraining« a complete different use case, so »1000« instead of »1 or 2« ?

Ah sorry, the semantics of `--freeze-backbone` are the same as
`--warmup` so in both cases the value is iterations, not whole epochs.
The granularity of epochs is insufficient for large datasets.
